tests/test_api.py

```diff
@@ -168,6 +168,8 @@ def assert_coverage(actual, expected):
     assert actual['source'].strip() == expected['source'].strip()
     assert actual['name'] == expected['name']
     assert actual['coverage'] == expected['coverage']
+    if 'branches' in actual and 'branches' in expected:
```

Comment:
I think this should fail if 'branches' is present in one but not both dicts. Could you make this:

    assert actual.get('branches') == expected.get('branches')